internal/parser/simple_parser_rules.go

```diff
@@ -1301,6 +1300,7 @@ func (p *simpleParser) parseExprBeginWithLiteral(literal token.Token, r reporter
 		returnExpr := p.parseExprRecursive(&ast.Expr{LiteralValue: literal}, r)
 		if returnExpr != nil {
 			expr = returnExpr
+			return
 		}
 		return nil
```

Comment:
```suggestion
		return returnExpr
```

Reply:
Well both are same aren't they :D

Reply:
That's why you should change it

Reply:
Can you request the change again? I can commit directly from here.

Reply:
I switched OS's thats why :D

Reply:
It's still there (?)